### DISCUSSION TITLE
Force a Channel to a Channel No based on part of Channel Name

### DISCUSSION THREAD

**Discussion by iceman73** [euzu/tuliprox#75] [General]
Like most users there are channels that come out weekly where the Channel Name and the Event are the channel name.

My client doesnt like that the channel name keeps changing. My provided invents a new channel num every time they update, so ther is a part of the channel title that is consistent XYZ01: Event Name. Event Name changes. Can I map XYZ: in title and always make it chno 5000, so forth for all these channels?

**Comment by euzu** [comment]
@iceman73 sorry i missed this post. 
You want to change the chno  of all channels matching a filter to 5000 ?
Or you want to change the title to  XYZ 5000 ?

Could you give me an example what you have and what you expect ?

**Reply by iceman73** [reply]
I was looking for a way to set something like this:
Channel Name Event01: Event at some time and description
to Channel 5001

I have a Bunch of Channels like this that I wanted to order number in some way. I just bit the bullet and created a single mapping for all 100 channels

**Comment by euzu** [comment]
@iceman73   is there a pattern in the name for all the channels ? Or do they have different names ?

I have implemented a new feature  which could be extended to fulfill  your needs? It is discussed here:
[counter](https://github.com/euzu/m3u-filter/discussions/85)

It does not fit your needs, but if we find a way to match your problem with some changes, then i could implement it.

**Comment by iceman73** [comment]
For the one I most want its always "ABC01: Title" The ABC is consistent, the 01, changes for each channel and Title changes on every event week.

This does look like it could fit what I wanted to do, I need to see if the channels come in order of 01-50 so they match

**Comment by euzu** [comment]
@iceman73 is this solved?

**Comment by iceman73** [comment]
Yes ty
